src/test/java/core/basesyntax/strategy/SupplyOperationHandlerTest.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.FruitDto;
+import core.basesyntax.model.Fruit;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class SupplyOperationHandlerTest {
+    private static OperationHandler handler;
+    private static Fruit apple;
+    private static Fruit banana;
+    private static Fruit orange;
+
+    @BeforeClass
+    public static void beforeClass() {
+        handler = new SupplyOperationHandler();
+        apple = new Fruit("apple");
+        banana = new Fruit("banana");
+        Storage.storage.put(apple, 35);
+        Storage.storage.put(banana, 8);
+    }
+
+    @Test
+    public void supplyHandler_addTransaction_Ok() {
+        FruitDto fruitDto = new FruitDto("s", "banana", 84);
+        handler.apply(fruitDto);
+        int expected = 92;
+        int actual = Storage.storage.get(banana);
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void supplyHandler_chekReturnValue_Ok() {
```

Comment:
```suggestion
    public void supplyHandler_checkReturnValue_Ok() {
```
what about `r` operation?